services/meta/data.go

```diff
@@ -1531,16 +1531,19 @@ func (leases *Leases) Acquire(name string, nodeID uint64) (*Lease, error) {
 
 // MarshalTime converts t to nanoseconds since epoch. A zero time returns 0.
 func MarshalTime(t time.Time) int64 {
-	if t.IsZero() {
+	if t.IsZero() || t.UTC() == time.Unix(0, 0).UTC() {
```

Comment:
That's what I thought, but that's not true.  `UnixNano` on a zero time is actually undefined.  

```go
package main

import "time"

func main() {
	var t time.Time
	println(t.String())
	i := t.UnixNano()
	println(i)
	t = time.Unix(0, i)
	println(t.String())
}
```

results: 

```
0001-01-01 00:00:00 +0000 UTC
-6795364578871345152
1754-08-30 22:43:41.128654848 +0000 UTC
```

Play Example: https://play.golang.org/p/P0s7lbZtnK

And the source:

https://golang.org/src/time/time.go?s=24148:24178#L827